rust/noosphere/src/ffi/context.rs

```diff
@@ -104,9 +104,47 @@ pub fn ns_sphere_fs_open(
     })
 }
 
+#[ffi_export]
+/// Access another sphere by a petname. The petname should be one that has been
```

Comment:
nit: add `\n\n` after the first sentence (e.g. having a whole row between the first sentence and the second sentence). This splits up the function documentation into "summary" and "detailed" information.